fusor-ember-cli/app/routes/review/installation.js

```diff
@@ -85,21 +85,22 @@ export default Ember.Route.extend(NeedsExistingManifestHelpers, {
 
     controller.set('validationErrors', []);
     controller.set('validationWarnings', []);
+    let self = this;
 
     if (!model.get('isStarted')) {
       // the PUT request from saveDeployment was firing too late and the server was syncing/validating stale data.
       // the model.save ensures the server has the most recent version of deployment before proceeding.
       controller.set('showSpinner', true);
-      this.validate()
-        .then(() => this.syncOpenStack())
-        .catch(error => {
-          console.log('error', error);
-          controller.set('errorMsg', error.jqXHR.responseText);
-          controller.set('showErrorMessage', true);
-        })
-        .finally(() => {
-          controller.set('showSpinner', false);
+      this.validate().then(function () {
+        return self.validateDiscoveredHostsPoweredOn().then(function () {
+            return self.syncOpenStack();
         });
```

Comment:
@cfchase, these are now nested to run in serial, rather than chained which runs in parallel.